Content/NoRenameWinClass.md

```diff
@@ -0,0 +1,25 @@
+# No Rename Win Class
+
+_NoRenameWinClass_ is a sandbox setting in [Sandboxie Ini](SandboxieIni.md). It specifies the window class names that should not be translated by Sandboxie.
+
+Examples:
+```
+   .
+   .
+   .
+   [DefaultBox]
+   NoRenameWinClass=ExampleWinClass
+   NoRenameWinClass=program.exe,*
+```
+
+The first setting tells Sandboxie to not translate _ExampleWinClass_ window class name, and also makes accessible to sandboxed programs, and goes a step further to disable a few other windowing-related Sandboxie functions. This may also cause the Sandboxie indicator [#] to not appear in window titles.
```

Comment:
```suggestion
The first setting tells Sandboxie to not translate _ExampleWinClass_ window class name by making it accessible to sandboxed programs, and goes a step further to disable a few other windowing-related Sandboxie functions. This may also cause the Sandboxie indicator [#] to not appear in window titles.
```